.github/workflows/validate.yml

```diff
@@ -44,16 +44,15 @@ jobs:
       - name: Install Dependencies
         run: pnpm i
 
-      - name: Build
-        run: pnpm build
-
       - name: Lint
         run: pnpm lint
```

Comment:
I thought lint depended on build because we want to lint generated files, e.g. icons?

Reply:
I don't remember that. But if lint depends on build, the relationship should be defined in the task graph.

Reply:
Yeah good point. So the cache should handle it and not rebuild. 👍